NuGet.config

```diff
@@ -2,21 +2,49 @@
+    <!--  End: Package sources from dotnet-corefx -->
+    <!--  Begin: Package sources from dotnet-core-setup -->
+    <add key="darc-int-dotnet-core-setup-0267ad0" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-0267ad09/nuget/v3/index.json" />
+    <!--  End: Package sources from dotnet-core-setup -->
+    <!--  Begin: Package sources from dotnet-extensions -->
+    <add key="darc-int-dotnet-extensions-4210e7d" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-extensions-4210e7d1/nuget/v3/index.json" />
+    <!--  End: Package sources from dotnet-extensions -->
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="dotnet3.1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1/nuget/v3/index.json" />
     <add key="dotnet3.1-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
     <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
   </packageSources>
   <disabledPackageSources>
-    <add key="darc-int-dotnet-extensions-ce485be" value="true" />
-    <add key="darc-int-dotnet-extensions-8be2e8d" value="true" />
-    <add key="darc-int-dotnet-extensions-77d62c5" value="true" />
+    <add key="darc-int-dotnet-extensions-bfff66d" value="true" />
+    <add key="darc-int-dotnet-extensions-57f40d5" value="true" />
+    <add key="darc-int-dotnet-extensions-8bac2ca" value="true" />
+    <add key="darc-int-dotnet-extensions-64d0e8d" value="true" />
+    <add key="darc-int-dotnet-extensions-1ec82f9" value="true" />
+    <add key="darc-int-dotnet-core-setup-5faea65" value="true" />
+    <add key="darc-int-dotnet-extensions-8e142d5" value="true" />
+    <add key="darc-int-dotnet-core-setup-d292495" value="true" />
+    <add key="darc-int-dotnet-extensions-77f5aa6" value="true" />
+    <add key="darc-int-dotnet-core-setup-08d8c2a" value="true" />
+    <add key="darc-int-dotnet-corefx-f571fb3" value="true" />
+    <add key="darc-int-dotnet-extensions-6022c35" value="true" />
+    <add key="darc-int-dotnet-core-setup-89c50a2" value="true" />
+    <add key="darc-int-dotnet-corefx-14aba64" value="true" />
```

Comment:
Same deal here @mmitche and @MattGal

Reply:
I fixed up some nuget.configs manually last month, this might have been me